scss/_settings_grid.scss

```diff
@@ -5,15 +5,29 @@
 $grid-max-width: 80rem !default;
 $grid-columns-small: 4 !default;
 $grid-columns-medium: 6 !default;
+$grid-columns-medium-new: 4 !default;
 $grid-columns: 12 !default;
+
 $threshold-4-6-col: $breakpoint-small !default;
+$threshold-4-small-4-med-col: $breakpoint-small !default;
 $threshold-6-12-col: $breakpoint-large !default;
+$threshold-4-med-12-col: $breakpoint-large !default;
 
 $grid-column-prefix: 'col-' !default;
+$grid-row-class: 'row' !default;
+$grid-row-4-cols-class: '#{$grid-row-class}--4-cols' !default;
 $grid-small-col-prefix: '#{$grid-column-prefix}small-' !default;
 $grid-medium-col-prefix: '#{$grid-column-prefix}medium-' !default;
 $grid-large-col-prefix: #{$grid-column-prefix} !default;
 
+// Common grid patterns
+$col-50-medium: calc($grid-columns-medium / 2);
+$col-50-medium-new: calc($grid-columns-medium-new / 2);
+
+$col-25: calc($grid-columns / 4);
+$col-50: calc($grid-columns / 2);
+$col-75: calc(($grid-columns / 4) * 3);
```

Comment:
This exposes a lot of variables to global SCSS scope.

If they are needed in a single file/mixin, would be good to keep it there.
If they need to be global, worth namespacing them a bit, at least with `$grid-` prefix.

Reply:
Moved these back to the grid pattern's scope